reverse-proxy.md

```diff
@@ -14,7 +14,64 @@ In order to run Nextcloud behind a reverse proxy, you need to specify the port t
 
 **Please note:** Since the Apache container gets spawned by the mastercontainer, there is **NO** way to provide custom docker labels or custom environmental variables for the Apache container. So please do not attempt to do this because you will fail! Only the documented way will work!
 
-### Caddy
+### Apache
+
+<details>
+
+<summary>click here to expand</summary>
+
+**Disclaimer:** It might be possible that the config below is not working 100% correctly, yet. Improvements to it are very welcome!
+
+Add this as a new Apache site config:
+
+```
+<VirtualHost *:80>
+    ServerName <your-nc-domain>
+    ServerAlias <your-nc-domain>
+
+    RewriteEngine On
+    RewriteCond %{HTTPS} off
+    RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+    RewriteCond %{SERVER_NAME} =<your-nc-domain>
+    RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,NE,R=permanent]
+</VirtualHost>
+
+<VirtualHost *:443>
+    ServerName <your-nc-domain>
+    ServerAlias <your-nc-domain>
+
+    # Reverse proxy
+    Protocols h2 http/1.1
+    RewriteEngine on
+    ProxyPreserveHost On
+    RewriteCond %{HTTP:Upgrade} websocket [NC]
+    RewriteCond %{HTTP:Connection} upgrade [NC]
+    RewriteRule ^/?(.*) "ws://locahost:11000/\$1" [P,L]
+    ProxyPass / http://locahost:11000/
+    ProxyPassReverse / http://locahost:11000/
+
+    # SSL
+    SSLProxyEngine On
+    SSLProxyVerify none
+    SSLProxyCheckPeerCN off
+    SSLProxyCheckPeerName off
+    SSLProxyCheckPeerExpire off
+    Header always set Strict-Transport-Security "max-age=15768000; includeSubDomains; preload"
+    <Files ".ht*">
+        Require all denied
+    </Files>
+    TraceEnable off
+    Include /etc/letsencrypt/options-ssl-apache.conf
+    SSLCertificateFile /etc/letsencrypt/live/<your-nc-domain>/fullchain.pem
+    SSLCertificateKeyFile /etc/letsencrypt/live/<your-nc-domain>/privkey.pem
```

Comment:
Maybe there should also be a note added if the user does not use letsencrypt

Reply:
Sounds good! :)

Reply:
or where he can read how he can use acme.sh / certbot to get certificates